### PR TITLE
Add channels to pointMatchClient

### PR DESCRIPTION
**Pull request by perlman**

### CODE REVIEW

**Comment by fcollman** [comment]
please add a test in the multi-channel component of the tests, where you intentionally load up a tilespec with differently ordered image channels and then have point matching fail when no option is specified, and then specify the option and have the point matching pass.

**Comment by perlman** [comment]
Is it really render-python's place to enforce render's behavior with regards to channel ordering? If ordering should matter, I think it should be fixed in render, not render-python.

**Comment by fcollman** [comment]
i wouldn't call this channel ordering, i would call it channel selection, we clearly need a way for render-python to specify channels.. you might argue that render should be modified to make it easier to get the default 'correct' answer, or at least only render the same channel, but render-python must ultimately put the channels in the tilespec in a consistent order.

**Comment by perlman** [comment]
How does this look?
Looking through the render code, channel order _is_ predictable, which is why it's a list and not a dictionary. If there is no mask/alpha on the first channel, it's all you get. Weird behavior occurs with missing data.

**Comment by fcollman** [comment]
doesn't look like you setup your fixtures correctly with creation of point match collections.

**Comment by fcollman** [comment]
i suspect that your point matching produces no point matches so it can't pull that there are no point matches and your local render might have some residual ones left over in that collection, so it returns no matches just fine.

**Comment by perlman** [comment]
Should I use a fixture to create an empty match? (It seems like other code using render-python just uses absence of matches as failure.)

File: renderapi/client/client_calls.py [hunk 1]
```diff
@@ -598,7 +608,13 @@ def pointMatchClient(stack, collection, tile_pairs,
         transforms that you had applied after it. default= None.
     excludeAllTransforms: bool
         alternative to normalizeForMatching which simply removes all transforms from the list.
-        default=False
+        default = False
+    stackChannels: str or None
+        If specified, option to select which channel is used for the stack.
+        default = None
+    stack2Channels: str or None
```
Comment (fcollman) [inline renderapi/client/client_calls.py]: what is the use case for doing cross channel point matches?

Comment (fcollman) [reply renderapi/client/client_calls.py]: if there isn't a compelling one I would just have this as "channels" like in the other api

Comment (perlman) [reply renderapi/client/client_calls.py]: Use case: "DAPI_1" vs "DAPI_2" vs "DAPI_3".

As for the test, given the test data, failing with different channels seemed like a reasonable test to show that the data are different.

Comment (fcollman) [reply renderapi/client/client_calls.py]: that is a good one:) ok

**Comment by fcollman** [comment]
i've asked @RussTorres  for his opinion, if he OKs it I'll merge.

**Comment by perlman** [comment]
(poke @RussTorres).

I think this the last bit of unmerged code from the actively used AT pipeline branches.

**Pull request merged**